adapters/amqp-vertx-base/src/main/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapter.java

```diff
@@ -105,14 +105,19 @@
 
     // These values should be made configurable.
     /**
-     * The minimum amount of memory that the adapter requires to run.
+     * The minimum amount of memory (bytes) that the adapter requires to run on a standard Java VM.
      */
-    private static final int MINIMAL_MEMORY = 100_000_000; // 100MB: minimal memory necessary for startup
+    private static final int MINIMAL_MEMORY_JVM = 100_000_000;
```

Comment:
Should we move these to the abstract adapter or even limit strategy class, so we don't have to repeat it for every adapter?

Reply:
That would, of course, make sense if the values would be the same for all adapters by definition. However, in general, I guess these are adapter (type) specific, aren't they?

Reply:
I supposed these were reasonable defaults for most of the adapters, that could be overridden if needed. Anyhow, the PR looks good to me.